src/SFA.DAS.EmployerIncentives.Web/Controllers/CancelApprenticeshipsController.cs

```diff
@@ -56,16 +57,21 @@ public async Task<IActionResult> Confirm(SelectApprenticeshipsRequest request)
 
         [HttpPost]
         [Route("{accountLegalEntityId}/application-cancelled")]
-        public async Task<IActionResult> Cancelled(SelectApprenticeshipsRequest request)
+        public async Task<IActionResult> Cancelled(SelectApprenticeshipsRequest request, string accountId)
         {
             if (!request.HasSelectedApprenticeships)
             {
                 return RedirectToAction("ListPaymentsForLegalEntity", "Payments", new { request.AccountId, request.AccountLegalEntityId });
             }
 
-            var apprenticeshipIncentivesToWithdraw = (await GetSelectViewModel(request.AccountId, request.AccountLegalEntityId)).ApprenticeshipIncentives.Where(a => a.Selected);
+            var apprenticeshipIncentivesToWithdraw = 
+                (await GetSelectViewModel(request.AccountId, request.AccountLegalEntityId))
+                .ApprenticeshipIncentives.Where(a => a.Selected).ToList();
 
-            await _apprenticeshipIncentiveService.Cancel(request.AccountLegalEntityId, apprenticeshipIncentivesToWithdraw);
+            var emailAddress = ControllerContext.HttpContext.User.FindFirst(c => c.Type.Equals(EmployerClaimTypes.EmailAddress))?.Value;
+
+            await _apprenticeshipIncentiveService.Cancel(request.AccountLegalEntityId, apprenticeshipIncentivesToWithdraw,
```

Comment:
The AC says that the email should be sent to the user that submitted the application for the cancelled apprentice - the logged in user might not be the person who originally submitted - worth clarifying

Reply:
SubmittedByEmail is on the Application record.

Reply:
What if they have left?

Reply:
It probably makes sense for the confirmation email to go to the person who actioned the cancellation, but the ticket doesn't state that at present.

Reply:
AC in tickets updated, approved